src/app/organizations/organization-details/manage-projects/manage-single-project/manage-single-project.component.ts

```diff
@@ -65,25 +63,13 @@ export class ManageSingleProjectComponent {
         );
 
         this.updateForm$ = this.project$.pipe(map(project => {
-            const dateFrom = this.datePipe.transform(project.start_date, 'mediumDate');
-            const dateTo = this.datePipe.transform(project.end_date, 'mediumDate');
-            const investFrom = this.currencyPipe.transform(project.min_per_user, 'EUR', 'symbol', '2.0');
-            const investTo = this.currencyPipe.transform(project.max_per_user, 'EUR', 'symbol', '2.0');
-            const expectedInvest = this.currencyPipe.transform(project.expected_funding, 'EUR', 'symbol', '2.0');
```

Comment:
As we talked, no need for having this in the form if the values won't be ever changed. This can be set in the template.